launchlib/config.go

```diff
@@ -69,6 +69,7 @@ type CustomLauncherConfig struct {
 	Env                     map[string]string          `yaml:"env"`
 	Experimental            ExperimentalLauncherConfig `yaml:"experimental"`
 	DisableContainerSupport bool                       `yaml:"dangerousDisableContainerSupport"`
+	HeapPercentage          *float64                   `yaml:"heapPercentage" validate:"gt=0,lte=100"`
```

Comment:
Does the `validate` block make `go` throw an error when it's out of this range? Should we add a test for that (maybe it's obvious/not needed)?

Reply:
Yeah it throws an error: https://pkg.go.dev/gopkg.in/validator.v2

No need to test this IMO